src/displayapp/screens/Notifications.h

```diff
@@ -33,31 +33,35 @@ namespace Pinetime {
 
         class NotificationItem {
         public:
+          NotificationItem(Pinetime::Controllers::AlertNotificationService& alertNotificationService,
+                           Pinetime::Controllers::MotorController& motorController,
+                           bool isTransition = false);
           NotificationItem(const char* title,
                            const char* msg,
                            uint8_t notifNr,
                            Controllers::NotificationManager::Categories,
                            uint8_t notifNb,
-                           Modes mode,
                            Pinetime::Controllers::AlertNotificationService& alertNotificationService,
-                           Pinetime::Controllers::MotorController& motorController);
+                           Pinetime::Controllers::MotorController& motorController,
+                           bool isTransition = false);
```

Comment:
isTransition is never used because using the simple constructor creates an empty screen regardless, but the separate constructor wouldn't be needed either if the previous suggestion is implemented.